### PR TITLE
Add tools owners to CODEOWNERS

### PR DESCRIPTION
**Pull request by JimSuplizio**
Add entries for code-owners-parser, github-event-processor and github-team-user-store,

### CODE REVIEW

**Comment by JimSuplizio** [comment]
Overriding check-enforcer as there's currently nothing that'll run that'll test a CODEOWNERS change.

**Comment by JimSuplizio** [comment]
Overriding check-enforcer as there's currently nothing that'll run that'll test a CODEOWNERS change outside of GitHub's validation which shows there are no errors.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -31,7 +31,10 @@
 ###########
 /tools/                                @azure/azure-sdk-eng
 /tools/check-enforcer/                 @praveenkuttappan @weshaggard
+/tools/code-owners-parser/             @konrad-jamrozik @JimSuplizio
 /tools/github-issues/                  @praveenkuttappan @weshaggard
+/tools/github-event-processor/         @JimSuplizio
+/tools/github-team-user-store/         @JimSuplizio
```
Comment (weshaggard) [inline .github/CODEOWNERS:37]:
```suggestion
/tools/github-event-processor/         @JimSuplizio @benbp 
/tools/github-team-user-store/         @JimSuplizio @weshaggard
```

**Review by weshaggard — APPROVED** [review]

**Pull request merged**